src/bin/Makefile

```diff
@@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
         show-alignments compile-questions cluster-phones \
         compute-wer compute-wer-bootci make-h-transducer \
```

Comment:
I think I used something like
ar d name_of_old_file.o

I got that called every time by re-defining one of the compilation targets
(I think this from makefiles/default_rules.mk):
$(LIBFILE): $(OBJFILES)
in the local Makefile

but I don't know from the top of my head.
y.

On Tue, Mar 15, 2016 at 9:14 PM, Kirill Katsnelson <notifications@github.com
> wrote:

> In src/bin/Makefile
> <https://github.com/kaldi-asr/kaldi/pull/579#discussion_r56269028>:
>
> > @@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
> >          show-alignments compile-questions cluster-phones \
> >          compute-wer compute-wer-bootci make-h-transducer \
>
> Thanks, and also, IIRC, something needs to be done to lm/kaldi-lm.a due
> to removed source files there, but I do not remember the incantation. Do
> you?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/579/files/ee02fd543e4531aa003cbf296fd6ba50f7d7ea12#r56269028>
>